docs/index.html

```diff
@@ -696,6 +701,26 @@ <h5>Sample E-Commerce Workflow</h5>
 back to the Express Carrier (14) well in advance of the goods arrival.
         </p>
       </section>
+      <section>
+        <h5>Software Bill of Materials</h5>
+        <dl class="termlist">
+          <p>
+            Software is widely built from open source and other types of components. A Software Bill of Material
```

Comment:
We could start with the "why" as opposed to the "what" . The why is that physical supply chains now depend on software supply chains. Being able to secure digital services has become a requirement for securing physical ones. 

It's become a national security issue, and that's why the US president admin has issed an executive order. One of the approaches for increased visibility and accountability in software is to provide a software bill of materials which allows consumers to understand what's in the software the same way food ingredients are labeled.